pkgs/games/polymc/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, mkDerivation, makeDesktopItem, fetchFromGitHub, cmake, jdk8, jdk, zlib, file, makeWrapper, xorg, libpulseaudio, qtbase, libGL, msaClientID ? "" }:
```

Comment:
This line is huge. Can you put one parameter per line? 

(Or at least break it in the 80-char rule...)

Reply:
done